internal/ignition/ignition.go

```diff
@@ -244,12 +199,6 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 		}
 
 	}
-
-	err = g.writeOperatorManifests(installerPath, envVars)
```

Comment:
what will compensate for the removal of this call ? 
the manifests need to be added to the cluster before line 222 is reached

Reply:
Have a look at https://github.com/openshift/assisted-service/blob/master/internal/cluster/cluster.go#L830 - this is a place in which manifest files are being added internally during host installation

Reply:
I forgot to add the call to the operator's API. Thanks for pointing it out.

Reply:
> Have a look at https://github.com/openshift/assisted-service/blob/master/internal/cluster/cluster.go#L830 - this is a place in which manifest files are being added internally during host installation

yeah, that's where I was going to add it :smile: